### PR TITLE
ansible: add pt_BR translation

### PR DESCRIPTION
**Pull request by deadpyxel**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by deadpyxel** [comment]
I think I'll need some help from the maintainers, as the CI seems to thing my translation in brazilian portuguese has typos in english.

File: pages.pt_BR/ansible.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+`ansible {{group}} -m ping`
+
+- Exibe fatos sobre um grupo de hosts invocando o módulo setup:
+
+`ansible {{group}} -m setup`
+
+- Executa um comando em um grupo de hosts invocando o módulo command com argumentos:
+
+`ansible {{group}} -m command -a '{{my_command}}'`
```
Comment (navarroaxel) [inline pages.pt_BR/ansible.md:21]: Please translate the placeholders, like `inventory_file`, `my_commad`, etc., for get this easier for non-English speakers.

Comment (deadpyxel) [reply pages.pt_BR/ansible.md]: I did the translation. I believe it should be correct now. Thanks for the review.

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**